src/time_integration/methods_2N.jl

```diff
@@ -119,13 +119,11 @@ function init(ode::ODEProblem, alg::SimpleAlgorithm2N;
     # initialize callbacks
     if callback isa CallbackSet
         foreach(callback.continuous_callbacks) do cb
-            error("unsupported")
+            throw(ArgumentError("Continuous callbacks are unsupported with the low-storage RK method (Carpenter, Kennedy 1994, 4th order, Solution 3)."))
```

Comment:
```suggestion
            throw(ArgumentError("Continuous callbacks are unsupported with the 2N storage time integration methods."))
```
I think `(Carpenter, Kennedy 1994, 4th order, Solution 3)` is just one example of a 2N storage time integration method.

Reply:
You are right. Sorry for this!